frontend/src/app/App.tsx

```diff
@@ -40,9 +40,11 @@ const App: React.FC = () => {
     loadError: fetchConfigError,
   } = useApplicationSettings();
 
+  const { clusterStorageClasses: appContextStorageClasses } = useClusterStorageClasses();
```

Comment:
Recommend not naming your variables for their use-cases and naming them for what they are. 

`appContextStorageClasses` sorta dictates its end-point... but in reality they are just `storageClasses`